src/UiElement/UiElementTrait.php

```diff
@@ -87,6 +99,32 @@ public function getIcon(): string
         return $this->metadata->getParameter('icon');
     }
 
+    /**
+     * @inheritdoc
+     */
+    public function getWireframe(): string
+    {
+        return $this->metadata->getParameter('wireframe');
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function getWireframeSvg(): string
+    {
+        try {
+            $code = $this->getWireframe();
+
+            if (empty($code)) {
+                return '';
+            }
+
+            return $this->twig->render(sprintf('@MonsieurBizSyliusRichEditorPlugin/Wireframe/%s.svg.twig', $code));
```

Comment:
**🎯 suggestion:** If $twig is nullable, we can have an empty string returned. WDYT?

Also, what about testing if the template exists as well as `%s.html.twig` and `%s.svg.twig`?
This way we can override the SVG by using our own HTML, like an `img` or anything else!‏